### PR TITLE
julian/Timeout to db calls

### PR DESCRIPTION
**Pull request by jtabalujan2**
Created utility function to have timeout capabilities in other areas based on a promise race

### CODE REVIEW

**Comment by rob2244** [comment]
@jtabalujan2 Looks great, did you run it with a short timeout to make sure it throws correctly?

**Comment by jtabalujan2** [comment]
> @jtabalujan2 Looks great, did you run it with a short timeout to make sure it throws correctly?

Hi @rob2244, I did but I might not have set up my DB correctly in the first place so I needed some help from the rest of you to double check that.

File: src/server/util.js [hunk 1]
```diff
@@ -4,6 +4,24 @@ function toTitleCase(str) {
...
+
+  //promise that rejects in <ms> milliseconds
+  let timeout = new Promise((resolve, reject) => {
+    let id = setTimeout(() => {
+      clearTimeout(id)
+      reject(`Timed out in ${msDelay} ms`)
+    }, msDelay)
+  })
+
+  //
```
Comment (laueian) [inline src/server/util.js]: Super small, but I think you mean't to delete these comment slashes

Comment (laueian) [reply src/server/util.js]: I realized I could just make a commit from github so I removed it

**Comment by laueian** [comment]
Change looks good! I'm sure we'll find ways to use the timeout util in other areas of the project.

**Pull request merged**